src/Microsoft.AspNet.Hosting/HostingServices.cs

```diff
@@ -69,21 +64,5 @@ public static IServiceCollection Create(IServiceProvider fallbackServices, Actio
             services.AddHosting(configuration);
             return services;
         }
-
-        // Manifest exposes the fallback manifest in addition to ITypeActivator, IHostingEnvironment, and ILoggerFactory
-        private class HostingManifest : IServiceManifest
```

Comment:
Lets do it now, so we want to nuke HostingServices.Create and AddHosting becomes internal/private/dead?

Also do we want to make HostingEngine a static method?  I don't think we really support anything other than new HostingEngine(fallback, configureServices).Start(context) now